src/js/generate-entries.sh

```diff
@@ -43,9 +43,4 @@ cd "$root"/tools/ts-generator/types &&
 cd "$root"/tools/ts-generator/rpc &&
   $py rpc_gen_js.py \
     --server-define-file "$root"/src/v/coproc/gen.json \
-    --output-file "$root"/src/js/modules/rpc/serverAndClients/server.ts
-
```

Comment:
Your combining different things into one commit. If you are changing the name of the server from process batch server to rpcServer , do that in a separate commit too

Reply:
Make sense, I'll do